Snakefile

```diff
@@ -1386,6 +1388,7 @@ rule prepare_energy_totals:
         countries=config["countries"],
         base_year=config["demand_data"]["base_year"],
         sector_options=config["sector"],
+        energy_demand=config["custom_data"]["sectoral_energy_demand"],
```

Comment:
If I get it properly, `energy_demand` stands in fact for `custom_energy_demand`. If that is the case, the parameter name must be revised as it can be misleading right now